mxcubeweb/core/components/user/usermanager.py

```diff
@@ -384,9 +384,6 @@ def _login(self, login_id: str, password: str):
 
         # Only allow remote logins with existing sessions
         if self.app.lims.lims_valid_login(login_res) and is_local_host():
-            if not self.app.lims.lims_existing_session(login_res):
```

Comment:
This would break the idea of that a session needs to exist to be able to login with proposal based logins. We need to check the login type in that case